recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -32,20 +39,19 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-        # There is unsureness if static linking before 1.3.4 did every work.
-        # If you need it, teak here, on Linux and OSX you might have success.
-        if tools.Version(self.version) < "1.3.4":
-            self.options.shared = True
 
     def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
         del self.settings.compiler.cppstd
         del self.settings.compiler.libcxx
-        if self.options.shared == False and self.settings.os == "Windows" and tools.Version(self.version) < "1.3.4":
-            raise ConanInvalidConfiguration("Static linking in Windows did not work before version 1.3.4")
+
+        if not self.options.shared and tools.Version(self.version) < "1.3.4":
```

Comment:
This never has been true, not even on Windows where the problem existed due to linke flags.

but I do not think it's worth to start a discussion about that

Reply:
Please, this PR has been way to hard as is!

Reply:
you did a awesome job Chris!

Reply:
Thank you ❤️